notebook/static/base/js/keyboard.js

```diff
@@ -225,6 +225,17 @@ define([
         }
         return dct;
     };
+    
+    ShortcutManager.prototype.get_shortcut_for_action_name = function(name){
```

Comment:
I think get_shortcut or get_action_shortcut is about the shortest you can get without making this function's name too general.